utils/validate-install-plans.js

```diff
@@ -0,0 +1,148 @@
+const path = require('path');
+const { readYamlFile, pathsToSanitize } = require('./helpers');
+const {
+  fetchPaginatedGHResults,
+  filterInstallPlans,
+} = require('./github-api-helpers');
+
+const {
+  fetchNRGraphqlResults,
+  translateMutationErrors,
+} = require('./nr-graphql-helpers');
+
+const url = pathsToSanitize[0];
+
+const VALIDATE_INSTALL_PLAN_MUTATION = `# gql 
+mutation (
+  $description: String!
+  $displayName: String!
+  $fallback: Nr1CatalogInstallPlanDirectiveInput
+  $heading: String!
+  $id: ID!
+  $primary: Nr1CatalogInstallPlanDirectiveInput!
+  $target: Nr1CatalogInstallPlanTargetInput!
+) {
+  nr1CatalogSubmitInstallPlanStep(
+    dryRun: true
+    installPlanStep: {
+      description: $description
+      displayName: $displayName
+      fallback: $fallback
+      heading: $heading
+      id: $id
+      primary: $primary
+      target: $target
+    }
+  ) {
+    installPlanStep {
+      id
+    }
+  }
+}
+`;
+
+const transformInstallPlanTarget = (target) => {
```

Comment:
Could we add the comment blocks for documentation on the functions here?

Reply:
yes of course!